components/Cases/CaseLink.jsx

```diff
@@ -6,6 +6,8 @@ const getLink = (recordId, { form_name_overall, ...caseFormData }) => {
     case 'API_Allocation':
     case 'API_Deallocation':
       return `/people/${caseFormData.mosaic_id}/allocations/${caseFormData.allocation_id}?recordId=${recordId}`;
+    case 'ASC_case_note':
+      return `/people/${caseFormData.mosaic_id}/case-notes-recording/${recordId}`;
```

Comment:
I think this should be more generic, like `/people/${caseFormData.mosaic_id}/records/${recordId}`, we should have custom case links only for records not created using our forms